caas/kubernetes/provider/modeloperator.go

```diff
@@ -770,3 +771,22 @@ func ensureExecRBACResources(objMeta meta.ObjectMeta, clock jujuclock.Clock, bro
 func modelOperatorConfigMapAgentConfKey(operatorName string) string {
 	return operatorName + "-agent.conf"
 }
+
+func (k *kubernetesClient) GetModelOperatorDeploymentImage() (string, error) {
+	api := k.client().AppsV1().Deployments(k.namespace)
+	res, err := api.Get(context.Background(), modelOperatorName, metav1.GetOptions{})
+	if k8serrors.IsNotFound(err) {
+		return "", errors.NewNotFound(err, "k8s")
```

Comment:
internalerrors.Errorf("deployment %q not found", modelOperatorName).Add(errors.NotFound)

Reply:
hi kelvin do we have internal/errors in 3.6?

Reply:
I mean the error msg can be improved.